topo/controller/faucet_structural.yaml

```diff
@@ -0,0 +1,45 @@
+dps:
+  sw1:
+    dp_id: 0x12345
+    interfaces:
+      1:
```

Comment:
These interface assignemnts don't make any sense to me, but that's no a big deal since it's easy to tweak later.  I don't understand how this is mapping out to what it would look like on a physical switch (if that was somewhat the intent).

Reply:
Removed interface 7. Tested on switch and didn't see problems.